README.md

```diff
@@ -1,15 +1,25 @@
-# OSDC website
-OSDC's website, built on GatsbyJS and ReactJS.
-<br>
-## [osdc.surge.sh](https://osdc.surge.sh/)
+# Open Source Developers Community (OSDC) Official Website
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/b0a6511e-d7f3-4d1b-9799-4a33cf97d4d9/deploy-status)](https://app.netlify.com/sites/osdc/deploys)
+
+Repository containing the source code of OSDC's website built on [GatsbyJS](http://gatsbyjs.org) and [ReactJS](https://reactjs.org) hosted on [Netlify](app.netlify.com).<br>
+
+Live at [osdc.netlify.com](https://osdc.netlify.com).🚀
+
+<p align="center"><img src="osdc.gif"></p>
 
 ## Setup
-`yarn`
 
-`npm install --global gatsby-cli`
+- Fork and then clone the repository.
+- Install [yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
+- Run `yarn` to install the necessary packages.
+- Run `gatsby develop` to run the website in development mode on `localhost:8000`.
```

Comment:
Need to install gatsby-cli before this, maybe add that too

Reply:
Apart from this everything else looks good. Please make the changes and we'll be good to go.

Reply:
Sure. Making the changes. I though gatsby-cli would be one of the packages installed by yarn. But it was just gatbsy.

Reply:
@saranshbarua PTAL. Done.

Reply:
Great. LGTM.